modules/core/src/main/scala/org/tessellation/infrastructure/rewards/Rewards.scala

```diff
@@ -74,10 +73,19 @@ object Rewards {
 
           val mintedAmount = getAmountByEpoch(epochProgress, config.rewardsPerEpoch)
 
-          allRewardsState
+          val rewards: F[SortedSet[RewardTransaction]] = allRewardsState
             .run(mintedAmount)
             .flatTap(validateState(mintedAmount))
             .map(toTransactions)
+
+          val oneTimeRewards = config.oneTimeRewards
+            .filter(_.epoch === epochProgress)
+            .map(otr => RewardTransaction(otr.address, otr.txnAmount))
+
+          // TODO should we handle case where Address already in rewards map and sum TransactionAmounts?
+          rewards.map(_.concat(oneTimeRewards))
```

Comment:
we don't need to sum tx amounts. it's fine to keep it as a separate reward transaction

Reply:
@marcinwadon If we keep it separate then that requires a change in the return type doesn't it? And that will have a ripple effect at the caller.

Reply:
we already have multiple rewards for the same address if that's what you mean

Reply:
@marcinwadon I misspoke -- it's a set so it would have to be same address for same amount. But I confirmed with Ryle this would be highly unlikely.